src/main/java/dao/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package dao;
```

Comment:
The FileWriter interface should be in a 'service' package or a subpackage of it, such as 'service.io', to follow the project structure guidelines.